lib/info.js

```diff
@@ -325,6 +325,8 @@ const VIDEO_EURL = 'https://youtube.googleapis.com/v/';
 const getVideoInfoPage = async(id, options) => {
   const url = new URL(`https://${INFO_HOST}${INFO_PATH}`);
   url.searchParams.set('video_id', id);
+  url.searchParams.set('c', 'TVHTML5');
+  url.searchParams.set('cver', '7.20190319');
```

Comment:
is it possible to get this number from one of the prev. fetched pages? not a big fan of hard coding version numbers... 😅

Reply:
Not sure, I got the information from @zerodytrash. StackOverflow answers also suggest `6.20180913` for the version
https://stackoverflow.com/questions/67615278/get-video-info-youtube-endpoint-suddenly-returning-404-not-found

I tried it with other random numbers and strings and it seems to work too so I guess it only has to be set to something.

Reply:
actually just searching the watch page gave me this
![image](https://user-images.githubusercontent.com/17046330/123009431-f0ab4280-d3bc-11eb-8a07-c9a1149dcd09.png)
`cver: 7.20190319` is over 1 year old...

Reply:
Should I change it to that value?

Reply:
setting `cver` to `2.20210621.02.00` seems to cause the issue all over again

Reply:
ok seems like to set `cver` to `2.20210621.02.00`,you need to set `c` to `WEB`. but when i did this it still caused 404s sometimes

Reply:
for now i am applying a local patch with `c` set to `TVHTML5` and `cver` set to `6.20180913`